Add og-image for better branding

surrealist opened the pull request:
All images designed by @MennStudio

Resolve #24

surrealist committed:
Add og-image for better branding

All images designed by @MennStudio
Resolve #24

aaronamm commented on config/config.yml:
```diff
@@ -8,6 +8,7 @@ metadata:
   siteImage:
   favicon: "/assets/favicon.png"
   themeColor: "#ff0000"
+  ogImage: "/assets/dotnetthailand-ogimage-dark.png"
```
@surrealist  Just curiosity, how the current system works without adding ogImage.
I wonder if we don't need to add ogImage here.

surrealist replied:
I pushed new commit with detailed explanation.

surrealist committed:
Update image path.

We have 2 places that can inject meta og:image to html.
First one is src/html.js that need metadata:ogImage
from config/default.js. And the second one is
src/component/Soe/Soe.js this component need
metadata:siteImage from config/config.yml

I choose the second way (Soe.js) only then our web page
will have only one meta tag for og:image

surrealist commented:
I've tested on my local machine already, when inspect the web page I found this tag:

```html
<meta property="og:image" content="/assets/dotnetthailand-ogimage-dark.png" data-react-helmet="true">
```

I think this should be enough to complete the issue.

aaronamm commented:
@surrealist Thanks for your contribution.
